pyplanet/apps/contrib/voting/__init__.py

```diff
@@ -530,6 +559,11 @@ async def vote_extend(self, player, data, **kwargs):
 
 		:param player: player requesting the vote
 		"""
+		extend_max = await self.setting_extend_max_amount.get_value()
+		if 0 < extend_max <= self.extend_current_count:
+			message = f'$i$f00Map has reached extend limit (max {extend_max})'
```

Comment:
Maybe change the error message to: ``$i$f00You're not allowed to extend the map, maximum $fff{extend_max}$f00 extends are allowed.`` ?

Reply:
matter of taste, be free to change.